packages/h5wasm/src/local/worker.ts

```diff
@@ -38,7 +38,6 @@ async function openFile(file: File): Promise<bigint> {
 
 async function closeFile(fileId: bigint): Promise<number> {
   const h5wasm = await h5wasmReady;
-  h5wasm.flush(fileId);
```

Comment:
While I'm at it, I remove those `flush` calls from both providers, as they are totally unnecessary.